Metalama.Framework.Engine/Pipeline/AspectPipelineResult.cs

```diff
@@ -53,6 +53,8 @@ public sealed class AspectPipelineResult
 
         public ImmutableDictionaryOfArray<Ref<IDeclaration>, AnnotationInstance> Annotations { get; }
 
+        public ImmutableArray<ValidatorInstance> Validators { get; }
```

Comment:
Collecting all declaration validators is useless. A `bool` property `HasDeclarationValidator` is enough.